lib/signs_ui_web/controllers/messages_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule SignsUiWeb.MessagesController do
+  use SignsUiWeb, :controller
+
+  alias SignsUi.Signs.Messages
+
+  def index(conn, _params) do
+    messages = Messages.list_messages()
+    render(conn, "index.html", messages: messages, token: get_csrf_token())
```

Comment:
As I mentioned in slack, I don't think we need to deal with this csrf token stuff, but we do need some sort of authentication. I think a hard coded key we pass as a header in the request should be fine.

Reply:
oh, right - i knew i forgot to remove something.  currently the index requires the basic auth stuff - i can add some key to the create side of it though